modules/settings/Settings.js

```diff
@@ -13,8 +13,15 @@ function getLocalStorage() {
 
     // eslint-disable-next-line no-invalid-this
     const global = typeof window === 'undefined' ? this : window;
+    let storage;
 
-    return global.localStorage;
+    try {
+        storage = global.localStorage;
```

Comment:
Also this Exception will be thrown every time getLocalStorage is called.
Probably not a big deal at this point, but seems suboptimal to me.